### DISCUSSION TITLE
程序过程中使用出现白屏，一直加载现象

### DISCUSSION THREAD

**Discussion by yumengmaomao** [AlistGo/alist#568] [Q&A]
最开始我使用的是win的服务器，当不在使用一段时间后，出现访问web端白屏，一直在加载的现象 webdav也是一直加载 ，需要重启程序 或者在管理页面点击清除缓存才可以。今天受不了，想是不是版本问题，于是更换服务器系统为centOS7使用一键安装脚本安装，还是出现这个问题，第一次打web端就是白屏无法加载，希望可以给一下解决办法

**Comment by yumengmaomao** [comment]
配置
![Screenshot_2022-02-19-20-04-29-381_com sonelli juicessh](https://user-images.githubusercontent.com/72592551/154800036-c50aed32-35d9-492b-a15d-39423ceea11a.jpg)
访问
![Uploading Screenshot_2022-02-19-20-05-06-967_mark.via.jpg…]()

**Comment by xhofe** [comment]
#227
